README.MD

```diff
@@ -1,28 +1,51 @@
-## javascript-sxt-sdk (v.2.0.6)
+## SxT-NodeJS-SDK (v.2.0.7)
+
 
 JavaScript(NodeJS) SDK for Space and Time Gateway (javascript version >= 19.8.0)
 
 ## Installation Instructions
 
+#### Installation using a package
+
+```sh
+npm install sxt-nodejs-sdk
+```
+
+or
+
+```sh
+yarn add sxt-nodejs-sdk
+```
+
+#### Working with source code
+
 _Note: Before running the code, rename `.env.sample` to `.env` and ensure that your credentials are setup in the `.env` file properly_
 
 ```sh
 npm install
 ```
 
+or
+
+```sh
+yarn
```

Comment:
Shouldn't this be ``yarn add``? Or Did ``yarn`` change recently and this command is enough?

Reply:
`yarn` command is equivalent to `npm install`